index.html

```diff
@@ -289,12 +289,12 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-      <div class="contact">
+      <section class="contact">
         <p class="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in</p>
         <form action="#" class="contact-form">
-          <input type="text" class="contact-name" placeholder="Full name" required>
+          <input type="text" class="contact-name" placeholder="Full name" maxlength="500" required>
           <input type="email" class="contact-email" placeholder="Email address" required>
-          <textarea name="text" cols="30" rows="10" class="contact-text" placeholder="Enter text here"></textarea>
+          <textarea name="text" cols="30" rows="10" class="contact-text" placeholder="Enter text here" maxlength="50" required></textarea>
           <button type="submit" class="contact-submit">Get in touch</button>
         </form>
         <div class="social-media">
```

Comment:
Hi @anagudelogu 💯 Good work working on the changes from the last reviewer, 

- Make sure the full name accepts a max of 50 characters, you put a maxlength of 500 characters.

- At the same point where the textarea should have 500 characters as maxlength, you put 50.

Kindly make this changes so you can continue whit your next project! 🥇